packages/popover/src/Popover.tsx

```diff
@@ -93,6 +93,14 @@ export interface PopoverProps {
    * default: false
    */
   adjustOnMutation?: boolean;
+
+  setAlignment?: (
+    alignment: Align,
+  ) => void | React.Dispatch<React.SetStateAction<string>>;
```

Comment:
Can we use `Align` instead of `string` as the type parameter to `React.SetStateAction`?

Reply:
👍